tests/Unit/Controllers/Store/BundleControllerTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+
+namespace Tests\Unit\Controllers\Store;
+
+use Auth;
+use App\Registration;
+use App\Bundle;
+use App\Centre;
+use App\CentreUser;
+use App\Voucher;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class BundleControllerTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected $centre;
+    protected $centreUser;
+
+    /** @var Registration $registration */
```

Comment:
Are these intermittent doc comments being injected by your  IDE? there doesn't seem to be consistency of use.

Reply:
Yes, it's phpdoc to tell the ide about objects it can't resolve.